Source/Particles/WarpXParticleContainer.H

```diff
@@ -13,7 +13,7 @@
 #include "WarpXParticleContainer_fwd.H"
 
 #include "Evolve/WarpXDtType.H"
-#include "SpeciesPhysicalProperties.H"
+#include "Particles/ParticleBoundaries.H"
```

Comment:
Accidentally removed in conflict resolution:
```suggestion
#include "Particles/ParticleBoundaries.H"
#include "SpeciesPhysicalProperties.H"
```